internal/server/node/property_value.go

```diff
@@ -108,12 +113,57 @@ func GetPropertyValuesHelper(
 	arcOut bool,
 ) (map[string][]*model.Node, error) {
 	rowList := bigtable.BuildPropertyValuesKey(dcids, prop, arcOut)
-	// Current branch cache is targeted on new stats (without addition of schema etc),
-	// so only use base cache data for property value.
-	//
-	// TODO(shifucun): perform a systematic check on current cache data and see
-	// if this is still true.
-	return readPropertyValues(ctx, store, rowList)
+	baseDataList, _, err := bigtable.Read(
+		ctx,
+		store.BtGroup,
+		rowList,
+		func(dcid string, jsonRaw []byte) (interface{}, error) {
+			var propVals model.PropValueCache
+			err := json.Unmarshal(jsonRaw, &propVals)
+			if err != nil {
+				return nil, err
+			}
+			return propVals.Nodes, nil
+		},
+		nil,
+		true, /* readBranch */
+	)
+	if err != nil {
+		return nil, err
+	}
+	result := map[string][]*model.Node{}
+	visited := map[string]map[string]struct{}{}
+	// Loop the import groups. They are ordered by the preferences. So only add
```

Comment:
sorry, could you point at where they are ordered?

Reply:
Order is in bigtable_import_groups.json now. When import groups are finalized, should need ordering when creating the table clients.